templates/master.yaml

```diff
@@ -0,0 +1,858 @@
+             --ciphertext-blob fileb://$encKey \
+             --output text \
+             --query Plaintext \
+           | base64 -d > $f
+           mv -f $f ${encKey%.enc}
+         done;
+         echo done.'
+
+    rkt rm --uuid-file=/var/run/coreos/decrypt-tls-assets.uuid || :
+
+- path: /etc/kubernetes/ssl/apiserver.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.APIServerCert}}
+
+- path: /etc/kubernetes/ssl/apiserver-ca.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.APIServerCACert}}
+
+- path: /etc/kubernetes/ssl/apiserver-key.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.APIServerKey}}
+
+- path: /etc/kubernetes/ssl/calico/client.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.CalicoClientCert}}
+
+- path: /etc/kubernetes/ssl/calico/client-ca.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.CalicoClientCACert}}
+
+- path: /etc/kubernetes/ssl/calico/client-key.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.CalicoClientKey}}
+
+- path: /etc/kubernetes/ssl/etcd/server.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.EtcdServerCert}}
+
+- path: /etc/kubernetes/ssl/etcd/server-ca.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.EtcdServerCACert}}
+
+- path: /etc/kubernetes/ssl/etcd/server-key.pem.enc
+  encoding: gzip+base64
+  content: {{.TLSAssets.EtcdServerKey}}
+
+coreos:
+  units:
+  - name: add-route-bridge.service
+    enable: true
+    command: start
+    content: |
+      [Unit]
+      Description=Add route to reach local network
+      Requires=network-online.target
+      After=network-online.target
+
+      [Service]
+      Type=one-shot
+      ExecStart=/usr/bin/ip route add ${HOST_PUBLIC_IP} dev eth0 via ${BRIDGE_IP}
+
+      [Install]
+      WantedBy=multi-user.target
+  - name: set-ownership-etcd-data-dir.service
+    enable: true
+    command: start
+    content: |
+      [Unit]
+      Description=Set ownership to etcd2 data dir
+      Wants=network-online.target
+      After=etc-kubernetes-data-etcd.mount
+
+      [Service]
+      Type=oneshot
+      RemainAfterExit=yes
+      TimeoutStartSec=0
+      ExecStart=/usr/bin/chown etcd:etcd /etc/kubernetes/data/etcd
+  - name: docker.service
+    enable: true
+    command: start
+    drop-ins:
+    - name: 10-giantswarm-extra-args.conf
+      content: |
+        [Service]
+        Environment="DOCKER_CGROUPS=--exec-opt native.cgroupdriver=systemd {{.Spec.Cluster.Docker.Daemon.ExtraArgs}}
+  - name: k8s-setup-network-env.service
+    enable: true
+    command: start
+    content: |
+      [Unit]
+      Description=k8s-setup-network-env Service
+      Wants=network-online.target docker.service
+      After=network-online.target docker.service
+
+      [Service]
+      Type=oneshot
+      RemainAfterExit=yes
+      TimeoutStartSec=0
+      Environment="IMAGE={{.Spec.Cluster.Docker.ImageNamespace}}/setup-network-env:{{.Spec.Cluster.Operator.NetworkSetupVersion}}
```

Comment:
FYI I think in the long run we want to pin the full image here. It will probably never change. See https://github.com/giantswarm/k8s-setup-network-environment.